perception/autoware_tensorrt_bevdet/include/autoware/tensorrt_bevdet/alignbev_plugin.hpp

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2021-2023, NVIDIA CORPORATION. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef AUTOWARE__TENSORRT_BEVDET__ALIGNBEV_PLUGIN_HPP_
+#define AUTOWARE__TENSORRT_BEVDET__ALIGNBEV_PLUGIN_HPP_
+
+#include <NvInfer.h>
+#include <unistd.h>
+
+#include <cmath>
+#include <iostream>
+#include <string>
+#include <vector>
+
+namespace
+{
+static const char * ALIGN_PLUGIN_NAME{"AlignBEV"};
```

Comment:
Consider replacing the `ALIGN_PLUGIN_NAME` and `ALIGN_PLUGIN_VERSION` macros with `constexpr` strings. Macros can lead to unexpected behavior and are generally discouraged in favor of `constexpr` variables for better compile-time safety and debugging.  This is especially true in header files to avoid potential naming conflicts and improve the clarity of the code.